Cubical/Data/Nat/Properties.agda

```diff
@@ -179,3 +180,32 @@ _choose_ : ℕ → ℕ → ℕ
 n choose zero = 1
 zero choose suc k = 0
 suc n choose suc k = n choose (suc k) + n choose k
+
+isEven'-suc : (n : ℕ) → isOdd' n → isEven' (suc n)
+isEven'-suc (suc zero) p = tt
+isEven'-suc (suc (suc n)) p = isEven'-suc n p
+
+isOdd'-suc : (n : ℕ) → isEven' n → isOdd' (suc n)
+isOdd'-suc zero p = tt
+isOdd'-suc (suc (suc n)) p = isOdd'-suc n p
+
+evenOrOdd : (n : ℕ) → isEven' n ⊎ isOdd' n
+evenOrOdd zero = inl tt
+evenOrOdd (suc zero) = inr tt
+evenOrOdd (suc (suc n)) = evenOrOdd n
+
+¬evenAndOdd : (n : ℕ) → ¬ isEven' n × isOdd' n
+¬evenAndOdd zero (p , ())
+¬evenAndOdd (suc zero) ()
+¬evenAndOdd (suc (suc n)) = ¬evenAndOdd n
+
+isPropIsEven' : (n : ℕ) → isProp (isEven' n)
```

Comment:
What about `isPropIsOdd'`?

Reply:
Since I define `isOdd' n` as `isEven' (suc n)`, it's the same thing. But I added it now for completeness.